src/category_theory/monad/limits.lean

```diff
@@ -62,12 +62,6 @@ def cone_point (D : J ⥤ algebra T) [has_limit.{v₁} (D ⋙ forget T)] : algeb
       erw [algebra.assoc (D.obj j), ←category.assoc, ←T.map_comp], },
   end }
 
-@[simp] lemma cone_point_a (D : J ⥤ algebra T) [has_limit.{v₁} (D ⋙ forget T)] :
-(cone_point D).a = limit.lift _ (
```

Comment:
Was there a reason the old lemma was stated in such a way, not referring to `c`?

Reply:
Not that I've been able to work out... My best guess is that I was struggling with something that turned out to not be an issue, but I never cleaned up at the end, and it slipped through the PR process.